packages/firestore/src/lite-api/vector_value.ts

```diff
@@ -19,7 +19,7 @@ import { isPrimitiveArrayEqual } from '../util/array';
 
 /**
  * Represents a vector type in Firestore documents.
- * Create an instance with {@link FieldValue.vector}.
+ * Create an instance with `{@link vector}`.
```

Comment:
I don't think this is right, there shouldn't be backticks? If you use the correct format, the resulting markdown files shouldn't have the word "@link" in them anymore, it should be a markdown link (brackets then parens)

Reply:
By the way, this is the way to have something that is both a link and code formatted - you have to use the code tags here in the comment, you can't use backticks:
```
<code>{@link vector}</code>
```